src/connection_string.ts

```diff
@@ -1282,7 +1282,9 @@ export const OPTIONS = {
   index: { type: 'any' },
   // Legacy Options, these are unused but left here to avoid errors with CSFLE lib
   useNewUrlParser: { type: 'boolean' } as OptionDescriptor,
-  useUnifiedTopology: { type: 'boolean' } as OptionDescriptor
+  useUnifiedTopology: { type: 'boolean' } as OptionDescriptor,
+  // MongoLogger
+  mongodbLogPath: { type: 'any' }
```

Comment:
This is added to allow hook the unified spec runner up with the client logger

Reply:
Willing to revert this and defer this change to NODE-4849

Reply:
This is fine, it doesn't exactly expose it as public API b/c the types aren't updated and we still have the symbol feat flag